cmd/argo/commands/clustertemplate/lint.go

```diff
@@ -0,0 +1,99 @@
+package clustertemplate
+
+import (
+	"context"
+	"fmt"
+
+	"os"
+	"path/filepath"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+
+	"github.com/argoproj/argo/cmd/argo/commands/client"
+	wfv1 "github.com/argoproj/argo/pkg/apis/workflow/v1alpha1"
+	cmdutil "github.com/argoproj/argo/util/cmd"
+	"github.com/argoproj/argo/workflow/validate"
+	"github.com/argoproj/argo/pkg/apiclient/clusterworkflowtemplate"
+)
+
+func NewLintCommand() *cobra.Command {
+	var (
+		strict bool
+	)
+	var command = &cobra.Command{
+		Use:   "lint (DIRECTORY | FILE1 FILE2 FILE3...)",
```

Comment:
can you copy-and-past cmd/argo/commands/lint.go? it has a simpler and more versatile implementation